functions/.eslintrc.json

```diff
@@ -82,7 +82,7 @@
 		"max-len": ["error", {
 			"code": 100
 		}],
-		"max-lines-per-function": ["error", 25],
+		"max-lines-per-function": ["error", 30],
```

Comment:
Why did you change this rule?

Reply:
I was checking out something and forgot revert back. I will revert back